### ISSUE TITLE
Allow admins to disable new team creation

### ISSUE THREAD

**Issue by mikemaddem** [NFM-Studios/project-olly#121]

**Comment by techlover1** [comment]
Could this be an env var?

**Comment by mikemaddem** [comment]
No. Unless you know of how to modify env vars from within the staff panel. Should be a simple boolean field in a model thats accessed from the staff panel

**Comment by techlover1** [comment]
Why does it need to be in the staff panel? I feel like it doesn't need to be changed very often (if ever)
